python/manylinux1/scripts/build_bison.sh

```diff
@@ -19,7 +19,7 @@
 BISON_VERSION=3.3.2
 NCORES=$(($(grep -c ^processor /proc/cpuinfo) + 1))
 
-wget https://ftp.gnu.org/gnu/bison/bison-${BISON_VERSION}.tar.gz
+curl -sL https://ftp.gnu.org/gnu/bison/bison-${BISON_VERSION}.tar.gz -o bison-${BISON_VERSION}.tar.gz
```

Comment:
Why do we need this change?

Reply:
there was an ssl error with wget
error - https://quay.io/repository/prudhvi/gandivacentos/build/f389a20c-3425-4f24-abb8-64f04123d12e

Reply:
The URL requires sign-in.
Can we see the log without sign-in? Or should we create an account on the site?

Reply:
/build_bison.sh
2/3/2020, 9:36:08 AM
2/3/2020, 9:36:08 AM ---> Running in 96fca1c5d77f
2/3/2020, 9:36:08 AM + BISON_VERSION=3.3.2
2/3/2020, 9:36:08 AM ++ grep -c '^processor' /proc/cpuinfo
2/3/2020, 9:36:08 AM + NCORES=3
2/3/2020, 9:36:08 AM + wget https://ftp.gnu.org/gnu/bison/bison-3.3.2.tar.gz
2/3/2020, 9:36:08 AM --2020-02-03 09:36:08-- https://ftp.gnu.org/gnu/bison/bison-3.3.2.tar.gz
2/3/2020, 9:36:08 AM Resolving ftp.gnu.org...
2/3/2020, 9:36:08 AM 209.51.188.20, 2001:470:142:3::b Connecting to ftp.gnu.org|209.51.188.20|:443...
2/3/2020, 9:36:08 AM connected.
2/3/2020, 9:36:08 AM OpenSSL: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
2/3/2020, 9:36:08 AM Unable to establish SSL connection.
2/3/2020, 9:36:08 AM Removing intermediate container 96fca1c5d77f

Reply:
Thanks.
It seems that `wget` is old and TLVv1.2 or later isn't supported.
Can we stop installing `wget` here https://github.com/apache/arrow/blob/master/python/manylinux1/Dockerfile-x86_64_base#L23 ?

Reply:
done. changed build_openssl script to use curl